atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitionsTest.java

```diff
@@ -39,4 +43,20 @@ public void compactionStrategiesShouldMatchWithOrWithoutPackageName() {
                         onlyClassName.compaction_strategy),
                 ColumnFamilyDefinitions.isMatchingCf(fullyQualified, onlyClassName));
     }
+
+
+    @Test
+    public void ifCfDefsHaveDifferentGcGraceSecondsIsMatchingCfReturnsFalse() {
+        CfDef clientSideTable = ColumnFamilyDefinitions.getCfDef(
+                "test_keyspace",
+                TableReference.fromString("test_table"),
+                AtlasDbConstants.GENERIC_TABLE_METADATA);
```

Comment:
yea if we change the default value to something that equals the value in this test (which we likely will not) the test will fail and someone will just have to change the test value, so it's probably fine as is. But I see your point.